tests/helpers/matchers.js

```diff
@@ -0,0 +1,24 @@
+/** Test whether something is a promise. Works across realms. */
+function toBeAPromise(actual) {
+  const pass = Object.prototype.toString.call(actual) === "[object Promise]";
+  return {
+    pass,
+    message: () => `expected ${this.utils.printReceived(actual)}${pass ? " not" : ""} to be a promise`,
+  };
+}
```

Comment:
I was winding up with `Promise` instances from a different realm than the test was running in, so I couldn’t just do `expect(somePromise).toBeInstanceOf(Promise)`.

I *suspect* that the promise we get back from dynamic `import()` calls is actually from somewhere deep in Jest’s machinery, which is running in a different Node `vm` than the test itself, but am not really sure.